Automattic/simplenote-ios
# Fix/build ITC lane

### Fix
This PR fixes the `build_and_upload_itc` lane adding the missing automation for certificates and profiles handling. 

### Test
1. Run `bundle exec fastlane build_and_upload_itc`.
2. Verify that `gym` finishes successfully and that the upload to iTC starts. It will fail because the version is already present, but this is fine as the current failing phase is `gym`. If iTC recognizes that the version is already present, it means that the build and signing processes finished successfully.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
